open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/properties/EngineHostServicesConfig.java

```diff
@@ -23,17 +24,21 @@
  *         server running the Governance Engine OMAS that provides configuration for the governance engines
  *         as well as the governance actions that drive the automated execution of governance requests.</li>
  *
- *     <li>An array of EngineServiceConfig properties, one for each engine service to run.</li>
+ *     <li>An array of Engine properties, one for each engine to run.</li>
```

Comment:
the comments above say "is in two parts" it is now in 3 parts I think - as there are 3 things in the list now.